Tests/SnapshotTestingTests/__Snapshots__/SnapshotTestingTests/testURLRequest.post-curl.txt

```diff
@@ -0,0 +1,6 @@
+curl \
+	--request POST \
+	--header "Accept: text/html" \
+	--data 'pricing[billing]=monthly&pricing[lane]=individual' \
+	--cookie "pf_session={"user_id":"0"}" \
```

Comment:
Ah yup, see this snapshot. I believe these quotes should be escaped!

Reply:
😬 Oops! Good job someone has eagle eyes! Sorry about that